.github/workflows/desktop-build.yml

```diff
@@ -244,16 +251,22 @@ jobs:
           BUILDTYPE: '${{matrix.type}}'
           MAKE_TEST: 1
           MAKE_PACKAGE: '${{matrix.make_package}}'
-          PACKAGE_SUFFIX: '-macOS-${{matrix.target}}'
-        # macOS runner actually have only 3 cores
-        # See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
-        run: .ci/compile.sh --server --parallel 3
+          PACKAGE_SUFFIX: '-macOS${{matrix.target}}_${{matrix.soc}}'
+        # macOS runner have 3 cores usually - only the macos-13 image has 4:
+        # https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
+        # https://github.com/actions/runner-images?tab=readme-ov-file#available-images
+        run: |
+          if [[ ${{ matrix.os }} == macos-13 ]]; then
+            .ci/compile.sh --server --parallel 4
```

Comment:
please put this in the build matrix instead

Reply:
You mean the comment/links on the available images?

Reply:
https://github.com/Cockatrice/Cockatrice/pull/5040/files#r1820632104

Reply:
I specifically mean the core count differentiation